airbyte-integrations/connectors/source-google-analytics-v4/source_google_analytics_v4/source.py

```diff
@@ -469,17 +442,34 @@ def get_refresh_request_params(self) -> Mapping[str, any]:
         }
         headers = {"kid": self.client_id}
         signed_jwt = jwt.encode(payload, self.client_secret, headers=headers, algorithm="RS256")
-        return {"grant_type": "urn:ietf:params:oauth:grant-type:jwt-bearer", "assertion": signed_jwt}
+        return {"grant_type": "urn:ietf:params:oauth:grant-type:jwt-bearer", "assertion": str(signed_jwt)}
```

Comment:
Since line 432 was removed, it doesn't seem to be using the method from super() in non-jwt cases... (ie when not using service account json?)

In those cases, it seems the grant_type should be simply a `refresh_token` value:
https://github.com/airbytehq/airbyte/blob/f88b8313a81da3c30a6daefc26aaacfad8332b5a/airbyte-cdk/python/airbyte_cdk/sources/streams/http/auth/oauth.py#L59

Is this making the connector supporting only the service account / jwt?

Reply:
The get_authenticator method contains the logic for choosing the authorization type, there is support for client and service authorization